src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+
+import Display from './Components/Display';
 import './App.css';
+import ButtonPanel from './Components/ButtonPanel';
 
 function App() {
   return (
     <div className="App">
-      Hello World
+      <Display />
+      <ButtonPanel />
     </div>
   );
```

Comment:
[REQUIRED] The `App.js` component should wrap it's children in react fragments
![image](https://user-images.githubusercontent.com/73254040/97767570-8ae31100-1ae2-11eb-8283-2c61b6f5fd27.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)